jdbc/src/main/scala/zio/sql/jdbc.scala

```diff
@@ -32,9 +38,18 @@ trait Jdbc extends zio.sql.Sql with TransactionModule with JdbcInternalModule wi
   def execute(delete: Delete[_]): ZIO[SqlDriver, Exception, Int] =
     ZIO.serviceWithZIO(_.delete(delete))
 
+  def executeBatch(delete: List[Delete[_]]): ZIO[SqlDriver, Exception, List[Int]] =
```

Comment:
Lets keep it consistent and call all the `batch` methods executeBatch, not only the one for delete

Reply:
You are right. Actually i wanted to name them all uniformly `executeBatch` 
The problem is  that does not compile.
The compiler complains about `double definition
def executeBatch(delete: List[Jdbc.this.Delete[_]]): zio.ZIO[Jdbc.this.SqlDriver,Exception,List[Int]] at line 41 and
[error] def executeBatch(update: List[Jdbc.this.Update[_]]): zio.ZIO[Jdbc.this.SqlDriver,Exception,List[Int]] at line 53
[error] have same type after erasure: (delete: List): zio.ZIO
` 
therefore  i chose different name  `executeBatchDelete, executeBatchInsert and executeBatchUpdate`